src/lib/components/ui/withForwardedRef.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default (WrappedComponent) => {
```

Comment:
I'm just wondering where to place this helper (wrapper) because it actually isn't a UI component. Maybe `src/lib/components/` would be fine?

Reply:
Never mind, I overlooked it's in the `ui` directory and not in `ui/withForwardedRef/` which disturbed me. It's fine the way it is.